### PR TITLE
Better score

### PR DESCRIPTION
**Pull request by amnona**
Add double click on term in enrichment to see heatmap of all annotations with this term
Fix interactive html export to contact new dbbact address and fix safari bug
Bugfix in qt5 gui when showing sample/feature details which are not str
Update plot_diff_abundance_enrichment to return a new experiment with terms as features, features as samples (so can see what bacteria are contributing to what term)

### CODE REVIEW

File: calour/database.py [hunk 1]
```diff
@@ -320,3 +320,22 @@ def enrichment(self, exp, features, *kargs, **kwargs):
         '''
         logger.debug('Generic function for enrichment')
         return None
+
+    def show_term_details(self, term, exp, features, *kargs, **kwargs):
```
Comment (RNAer) [inline calour/database.py]: kargs -> args

File: calour/export_html_template.html [hunk 2]
```diff
@@ -61,9 +61,9 @@
       var yaxiswidth = 100;
       var xaxisheight = 20;
 
-      // fix for safari image smoothing interpolation - we want it pixelated!
-      var ctx = document.getElementById('canvas').getContext('2d');
-      ctx.webkitImageSmoothingEnabled = false;
+      // // fix for safari image smoothing interpolation - we want it pixelated!
```
Comment (RNAer) [inline calour/export_html_template.html]: could you add comments? or just deleted it if not relevant any more.

File: calour/heatmap/plotgui_qt5.py [hunk 3]
```diff
@@ -388,7 +388,13 @@ def enrichment(self):
                 else:
                     ccolor = 'red'
                 cname = cres['term']
-                listwin.add_item('%s - effect %f, pval %f ' % (cname, cres['odif'], cres['pvals']), color=ccolor)
+                dblclick_data = {}
```
Comment (RNAer) [inline calour/heatmap/plotgui_qt5.py]: add short description of the behavior?

File: calour/database.py (hunk 1 above)
Comment (amnona) [reply calour/database.py]: fixed

File: calour/export_html_template.html (hunk 2 above)
Comment (amnona) [reply calour/export_html_template.html]: not relevant. removed

File: calour/heatmap/plotgui_qt5.py (hunk 3 above)
Comment (amnona) [reply calour/heatmap/plotgui_qt5.py]: fixed

**Comment by amnona** [comment]
fixed all comments :)

**Pull request merged**